src/sentry/static/sentry/app/utils/mergeRefs.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * Combine refs to allow assignment to all passed refs
+ */
+export default function mergeRefs<T = any>(
+  refs: Array<React.MutableRefObject<T> | React.LegacyRef<T> | undefined>
+): React.RefCallback<T> {
+  return value => {
+    refs.forEach(ref => {
+      if (typeof ref === 'function') {
+        ref(value);
+      } else if (ref !== null && ref !== undefined) {
```

Comment:
should we remove the` else if`?

Reply:
Why?

Reply:
I thought we had agreed to avoid using `else if`. We could right this part like:

```suggestion
      return;
      } 
      
     if (defined(ref)) {
       (ref as React.MutableRefObject<T | null>).current = value;
       return;
     }
     
     return;
```

Reply:
I think it's OK in this context. I'm more concerned about usage where the conditionals are deeply nested or very long.